README.md

```diff
@@ -236,9 +236,9 @@ typedef enum : NSUInteger {
 
 + (NSValueTransformer *)updatedAtJSONTransformer {
     return [MTLValueTransformer reversibleTransformerWithForwardBlock:^(NSString *str) {
-        return [self.dateFormatter dateFromString:str];
+        return [self.class.dateFormatter dateFromString:str];
```

Comment:
That's because it's called from an instance method, where self is an instance of `GHIssue`. In method you've changed, self is the class because it's inside a class method.